xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/ScopeNotificationFilter.java

```diff
@@ -147,7 +147,7 @@ public String queryFilterAND(DocumentReference user, NotificationFormat format,
     @Override
     public Map<String, Object> queryFilterParams(DocumentReference user, NotificationFormat format)
     {
-        Map<String, Object> params = new HashedMap();
+        Map<String, Object> params = new ConcurrentHashMap<>();
```

Comment:
At some point, I had errors when accessing this HashMap ; now, it seems that we don’t need the ConcurrentHashMap anymore ; will change that :)